arm-asazure/2016-05-06/swagger/asazure.json

```diff
@@ -418,6 +430,7 @@
       "in": "path",
       "required": true,
       "type": "string",
+      "pattern": "^[-\\w\\._]+$",
```

Comment:
@amarzavery We should standardize this and make it part of good patterns/the skeleton Swagger file I'm working on.